src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentStatusFields.scala

```diff
@@ -16,7 +16,11 @@ object ConsignmentStatusFields {
   val mutationFields: List[Field[ConsignmentApiContext, Unit]] = fields[ConsignmentApiContext, Unit](
     Field("markUploadAsCompleted", OptionType(IntType),
       arguments = ConsignmentIdArg :: Nil,
-      resolve = ctx => ctx.ctx.consignmentStatusService.setUploadConsignmentStatusValueToComplete(ctx.arg(ConsignmentIdArg)),
+      resolve = ctx => ctx.ctx.consignmentStatusService.updateConsignmentStatus(
+        ctx.arg(ConsignmentIdArg),
```

Comment:
Hardcoding these values should be a temporary measure. We should:
1. create new generic mutation: `updateConsignmentStatus` which takes the relevant arguments
2. commit these changes
3. move any clients using the old mutation: `markUploadAsCompleted`
4. remove the old mutation